client/ayon_nuke/api/plugin.py

```diff
@@ -73,18 +74,21 @@ class NukeCreatorError(CreatorError):
 class NukeCreator(NewCreator):
     selected_nodes = []
 
-    def pass_pre_attributes_to_instance(
+    def _pass_pre_attributes_to_instance(
         self,
         instance_data,
         pre_create_data,
         keys=None
     ):
-        if not keys:
-            keys = pre_create_data.keys()
-
+        keys = keys or pre_create_data.keys()
```

Comment:
Maybe we should allow empty `keys`?
```suggestion
        if keys is None:
            keys = pre_create_data.keys()
```